src/components/encoding-pane/filter-shelf.tsx

```diff
@@ -6,13 +6,19 @@ import * as React from 'react';
 import * as CSSModules from 'react-css-modules';
```

Comment:
Let's rename this file and component to filter-pane.

Since you have shelf as a shelf for single filter, calling the grouping of multiple shelves a shelf too is a bit confusing :)